ilc/server/tailor/factory.js

```diff
@@ -7,15 +7,14 @@ const logger = require('../logger');
 const Tailor = require('tailorx');
 const fetchTemplate = require('./fetch-template');
 const Router = require('./server-router');
-const registryService = require('../registry/factory');
 const filterHeaders = require('./filter-headers');
 const errorHandlingService = require('../errorHandler/factory');
 const errorHandlerSetup = require('./error-handler');
 const fragmentHooks = require('./fragment-hooks');
 const ConfigsInjector = require('./configs-injector');
 const processFragmentResponse = require('./process-fragment-response');
 
-module.exports = function (cdnUrl, nrCustomClientJsWrapper = null) {
+module.exports = function (cdnUrl, nrCustomClientJsWrapper = null, registryService) {
```

Comment:
I guess `registryService` should be the first param because `nrCustomClientJsWrapper` is `null` by default

Reply:
moreover, imho it's better to use object(e.g `{cdnUrl, nrCustomClientJsWrapper = null, registryService}`) here, since we have too many params, and some of them are optional

Reply:
moreover, imho it's better to use object(e.g {cdnUrl, nrCustomClientJsWrapper = null, registryService}) here, since we have too many params, and some of them are optional

Reply:
Changed args order. We can use obj syntax later IMO